src/dao/pages/utils.ts

```diff
@@ -181,3 +182,40 @@ export const shortEnglishHumanizer = humanizeDuration.humanizer({
     },
   },
 });
+
+export function filterDAOs(
```

Comment:
instead of `utils` I dont think we have time but we can also use a separate UI hook .

Reply:
I agree. given the time constraint, lets leave this function in Utils for now.